### DISCUSSION TITLE
get_products

### DISCUSSION THREAD

**Discussion by aashishsaini75** [sergioteula/python-amazon-paapi#68] [Help] [answered]
@sergioteula I think something is not good with the get_products function, I am passing a list of asins in function to get the price value and also getting the response but the response is not returned in a order of list i am passing like the product[0].prices.price.value response should be the result of first asin of a list but it returned the value randomly of any of the asins.
See the attached images to get better understanding
![image](https://user-images.githubusercontent.com/12483054/121492812-d48ac700-c9f4-11eb-8428-618543280170.jpg)

**Comment by drstevendev** [comment]
please remove your API keys

**Comment by sergioteula** [comment]
That's so strange, I will test it and try to solve it for future updates in case there's an issue.

**Comment by sergioteula** [comment, marked as answer]
This issue should be solved with the last version.
